lib/defaults/defaults.go

```diff
@@ -836,4 +836,7 @@ const (
 	// IAMInviteTokenName is the name of the default Teleport IAM
 	// token to use when templating the script to be executed.
 	IAMInviteTokenName = "aws-discovery-iam-token"
+	// InstallerScriptName is the name of the by default populated,
+	// EC2 installer script
+	InstallerScriptName = "installer"
```

Comment:
Should we call it `default-installer` so it's clear that it's a default script?
```suggestion
	InstallerScriptName = "default-installer"
```